README.md

```diff
@@ -82,8 +82,8 @@ somecommand | upload
 somecommand | upload myfile.txt
 ```
 
-# API Documentation
-Want to make something that works with your uploader? You're in luck! We've got [API documentation](API.md).
+# Uploading with API
```

Comment:
Perhaps 'Uploading directly to your site'?

Reply:
I feel like that's a bit wordy. How about "Direct Uploading"?